scripts/travis/mode_tests-default.sh

```diff
@@ -41,6 +41,12 @@ if command -v docker > /dev/null ; then
   export LISK_ENABLED=1
   fold_end
 
+  fold_start "cosmos-start"
+  ./scripts/cosmos/clean.sh
```

Comment:
Do we need this call/script? This is done in start.sh automatically now, right?

Reply:
Ah sorry, thought I'd deleted it.